crates/autopilot/Cargo.toml

```diff
@@ -21,7 +21,7 @@ chrono = { version = "0.4", default-features = false, features = ["clock"] }
 clap = { version = "3.1", features = ["derive", "env"] }
 contracts = { path = "../contracts" }
 database = { path = "../database" }
-ethcontract = { version = "0.19.0", default-features = false }
+ethcontract = { git = "https://github.com/gnosis/ethcontract-rs.git", branch = "add-blockhash-to-event-filter", default-features = false }
```

Comment:
Is this included in v0.20.0?

Reply:
No, there is a PR https://github.com/gnosis/ethcontract-rs/pull/830 so will need to create v0.21.0